index.js

```diff
@@ -59,26 +59,49 @@ class SteamAuth {
         const response = await axios.get(
           `https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v0002/?key=${this.apiKey}&steamids=${steamId}`
         );
+
         const players =
           response.data &&
           response.data.response &&
           response.data.response.players;
 
-        if (players && players.length > 0) {
+        if (players && players.length > 0)
+        {
           // Get the player
           const player = players[0];
 
+          const AnimatedAvatar = await axios.get(
+            `https://api.steampowered.com/IPlayerService/GetAnimatedAvatar/v1/?key=${this.apiKey}&steamid=${steamId}`
+          );
+
+          const Animated = 
+            AnimatedAvatar.data &&
+            AnimatedAvatar.data.response &&
+            AnimatedAvatar.data.response.avatar;
+
+          const AvatarFrame = await axios.get(
+            `https://api.steampowered.com/IPlayerService/GetAvatarFrame/v1/?key=${this.apiKey}&steamid=${steamId}`
+          );
+
+          const Frame =
+            AvatarFrame.data &&
+            AvatarFrame.data.response &&
+            AvatarFrame.data.response.avatar_frame;
```

Comment:
Nit: Variable naming uses `camelCase` style elsewhere.

```suggestion
          const animatedAvatar = await axios.get(
            `https://api.steampowered.com/IPlayerService/GetAnimatedAvatar/v1/?key=${this.apiKey}&steamid=${steamId}`
          );

          const animated = 
            animatedAvatar.data &&
            animatedAvatar.data.response &&
            animatedAvatar.data.response.avatar;

          const avatarFrame = await axios.get(
            `https://api.steampowered.com/IPlayerService/GetAvatarFrame/v1/?key=${this.apiKey}&steamid=${steamId}`
          );

          const frame =
            avatarFrame.data &&
            avatarFrame.data.response &&
            avatarFrame.data.response.avatar_frame;

```